tests/pytest/addons.py

```diff
@@ -1,6 +1,29 @@
+import os
+import subprocess
+
 import pytest
 
 
+def _which(command):
```

Comment:
Can we use [shutil.which](https://docs.python.org/3/library/shutil.html#shutil.which) here?

Reply:
tested and switched out